TradeItIosTicketSDK2Tests/TradeItAccountManagementViewControllerSpec.swift

```diff
@@ -22,7 +22,7 @@ class TradeItAccountManagementViewControllerSpec: QuickSpec {
 
                 linkedBrokerManager = FakeTradeItLinkedBrokerManager()
                 TradeItLauncher.linkedBrokerManager = linkedBrokerManager
-                linkBrokerUIFlow = FakeTradeItLinkBrokerUIFlow(linkedBrokerManager: linkedBrokerManager)
+                linkBrokerUIFlow = FakeTradeItLinkBrokerUIFlow()
```

Comment:
I made this change a while ago but never updated the tests. They were not required in this classes so I could remove them.